src/app/page.tsx

```diff
@@ -42,9 +42,7 @@ export default function Home() {
         <div className='fixed-test' style={{ '--color': 'rgba(207, 23, 23, 0.246)' } as React.CSSProperties} />
         <div className='fixed-test' style={{ '--color': 'rgba(48, 207, 23, 0.271)' } as React.CSSProperties} />
         <div className='fixed-test' style={{ '--color': 'rgba(207, 23, 87, 0.236)' } as React.CSSProperties} />
-        <div className='fixed-test last' style={{ '--color': 'rgba(207, 23, 87, 0.236)' } as React.CSSProperties}>
-          <Footer />
```

Comment:
@aperdomoll90 - If we are adding it to the layout we should not need to add Footer explicitly into each page no?

Reply:
thats removing it from where it was